<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LibraryEmployeeInterface</class>
 <widget class="QDialog" name="LibraryEmployeeInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>191</width>
    <height>212</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/bleu/bleu.jpg);</string>
  </property>
  <widget class="QLineEdit" name="nameLineEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/white/white.jpg);</string>
   </property>
   <property name="placeholderText">
    <string>name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="idLineEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/white/white.jpg);</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::EchoMode::Password</enum>
   </property>
   <property name="placeholderText">
    <string>id</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/white/white.jpg);</string>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QLabel" name="statusLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/bleu/bleu.jpg);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
